internal/commands/run.go

```diff
@@ -33,28 +33,27 @@ const (
 	nameKindReference
 )
 
-const longDescription = `Install an application.
-By default, the application definition in the current directory will be
-installed. The APP_NAME can also be:
+const longDescription = `Run an application.
+By default, the application definition in the current directory will be ran. The APP_NAME can also be:
 - a path to a Docker Application definition (.dockerapp) or a CNAB bundle.json
 - a registry Application Package reference`
 
-const example = `$ docker app install myapp.dockerapp --name myinstallation --target-context=mycontext
-$ docker app install myrepo/myapp:mytag --name myinstallation --target-context=mycontext
-$ docker app install bundle.json --name myinstallation --credential-set=mycredentials.yml`
+const example = `$ docker app run myapp.dockerapp --name myinstallation --target-context=mycontext
```

Comment:
The examples are not up-to-date: we should remove the `bundle.json` one and the `.dockerapp` one